src/nccl_ofi_rdma.c

```diff
@@ -5345,15 +5331,27 @@ static int connect(nccl_net_ofi_ep_t *base_ep,
 
 		comm_state->stage = COMM_RECV_CONN;
 
-	case COMM_RECV_CONN:
+	case COMM_RECV_CONN:;
```

Comment:
stray `;`

Reply:
That was just C being difficult, but fixed.